happi/cli.py

```diff
@@ -102,13 +105,58 @@ def happi_cli(args):
                 return
             if value.replace('.', '').isnumeric():
                 logger.debug('Changed %s to float', value)
-                value = float(value)
-            client_args[criteria] = value
+                value = str(float(value))
 
-        results = client.search(**client_args)
-        if results:
+            if criteria == 'z' and ',' in value:
+                start = None
+                end = None
+                z_is_range = True
+                try:
+                    start, end = value.replace(
+                        '[', '').replace(']', '').split(',')
```

Comment:
It's probably ok in cases like this to only accept one form of input- no explicit need to support brackets in addition to no brackets (despite me suggesting it as an option earlier). Most users will soon forget that `z=[0, 250]` is even an option. Since you already went through the trouble though, may as well keep it in.

Reply:
I took this option out, I think you are right that soon people will forget about `[0,250]` being an option - also because i took it out made things a bit cleaner.... please let me know if I should add it back in